algebra/src/main/scala/endpoints/algebra/Urls.scala

```diff
@@ -97,6 +97,9 @@ trait Urls {
   /** An URL path carrying an `A` information */
   type Path[A] <: Url[A]
 
+  // This should remove false positive error in Intellij when using path as an url
+  implicit def pathToUrl[A](path: Path[A]): Url[A] = path
```

Comment:
Yes, this would be the best solution, but I don't know if that's gonna happen anytime soon, and this error may be really confusing for the users.